src/components/Monitoring/index.tsx

```diff
@@ -37,11 +37,11 @@ const useStyles = makeStyles((theme: Theme) => ({
   titleRootContainer: {
     display: 'flex',
     alignItems: 'center',
+    paddingBottom: theme.spacing(4),
```

Comment:
ideally margin to allow for space between components, padding if we want space inside the component so we can draw borders etc..

Reply:
made the change. Thanks for the feedback!